package.json

```diff
@@ -17,7 +17,7 @@
     "highcharts": "11.2.0",
     "loglevel": "1.8.1",
     "primeicons": "^6.0.1",
-    "primereact": "^10.3.0",
+    "primereact": "^10.2.1",
```

Comment:
If we want to pin it to 10.2 this should be `~10.2.1` (`~` means in the range of the minor version, `^` is in the range of the major version, so a lockfile update could update it)

Reply:
Ah, right. I always forget how those work.